README.adoc

```diff
@@ -109,6 +109,11 @@ This maven plugin simplifies the development of HiveMQ extensions as it provides
 |hivemq.jar
 |Name of the HiveMQ Jar-file
 
+|includedResource
```

Comment:
should this be named includeResources ?
since it includes a folder (multiple files)

Reply:
yeah thats better